Dependencies/xr/Source/OpenXR/SceneUnderstanding.cpp

```diff
@@ -0,0 +1,497 @@
+#define NOMINMAX // avoid including macro max definitions in favor of limit max definitions

```

Comment:
nit: Wrap this in an `#ifndef NOMINMAX` to avoid problems if someone ever adds this to a precompiled header or something